File: sem-3/python/ass1/Q10.py

# 10. Write a program to check whether input number is divisible by 3 or 5 or both.


n = int(input("Enter a number: "))

if n % 3 == 0 and n % 5 == 0:
    print(f"{n} is divisible by both 3 and 5")
elif n % 3 == 0:
    print(f"{n} is divisible by 3")
elif n % 5 == 0:
    print(f"{n} is divisible by 5")
else:
    print(f"{n} is not divisible by 3 or 5")

File: sem-3/php/ass1/Q10.php
<!-- Write a PHP program to create a simple calculator that can accept two numbers and perform
  operations like add, subtract, multiplication, and divide. (Using switch statement) (HTML + PHP) -->

<body>
<form method="post">
<input type="number" name="n1" placeholder="number">
<input type="number" name="n2" placeholder="number2">

    <label><input type="radio" name="op" value="+"> + (Add)</label>
    <label><input type="radio" name="op" value="-"> - (Subtract)</label>
    <label><input type="radio" name="op" value="*"> * (Multiply)</label>
    <label><input type="radio" name="op" value="/"> / (Divide)</label><br><br>

<button type="submit">submit</button>

</form>


<?php

if($_POST){
    $n1 = $_POST['n1'];
    $n2 = $_POST['n2'];
    $op = $_POST['op'];

    switch($op){
        case '+': echo ($n1 + $n2); break;
        case '-': echo ($n1 - $n2); break;
        case '*': echo ($n1 * $n2); break;
        case '/': echo ($n1 / $n2); break;
        default: echo 'pls select an opearator';
    }
}
?>

</body>